features/fork.feature

```diff
@@ -34,9 +34,12 @@ Feature: hub fork
         json :name => 'dotfiles', :owner => { :login => 'mislav' }
       }
       """
-    And I successfully run `git remote rename origin upstream`
     When I successfully run `hub fork --remote-name=origin`
-    Then the output should contain exactly "new remote: origin\n"
+    Then the output should contain exactly:
+      """
+      renaming origin as upstream
```

Comment:
Could the message say `renaming existing "origin" remote to "upstream"` so that it's more readable and clearer as to what's happening?